### PR TITLE
Transition to java 11+

### PR DESCRIPTION
**Pull request by jarrodgreene**
Removes/reworks functions deprecated in java 9, including the change
from system class loader no longer being a url class loader,
and asm library had to be updated

Now works with tests passing with both java 8 and 15,
but ant needs to be updated to do installer build

There are also a small number of formatting changes to match surrounding code

### CODE REVIEW

**Comment by timpokorny** [comment]
@jarrodgreene - Should we hold off on merging this until we've figured out and tested the examples and the C++ side as well?

**Comment by jarrodgreene** [comment]
Yeah that's probably best

**Review by timpokorny — CHANGES_REQUESTED** [review]
Looks good generally, just a couple of things that I think we should resolve now that I look at it.

Some comments in-line, but to add to those:

  * I have pushed a commit to the branch that patches Ant so that it has the same changes as the old v1.8.4 version. Looking at the way it sets up, I think we should roll back some of the changes to support vanilla ant. Now that the patched version of Ant 1.10.9 is on the branch, hopefully (fingers crossed) the old patch just works and we can roll back all of the little changes that had to manually prefix task names.
  * Need some sort of replacement for `verifyJdk`, even if we re-write the task to be able to verify 11+ JDKs as separate from 1.x style distributions. The code for CppTask is located at `github.com:timpokorny/cpptask.git`. It might be worth trying to update the root task.
  * Some hardcoded paths to deal with
  * Should defaults for byte code compatibility to v11 rather than v15.

The only other change I've made to note is that I've removed the Ant manual. It accounted for about 45mb of the 50mb that the weight of the distribution was. That change was made with the patched `ant.jar` and should be on the `java11` branch now.

File: codebase/profiles/linux/hla13.xml [hunk 1]
```diff
@@ -121,8 +124,8 @@
 		      failifexecutionfails="false"
 		      resultproperty="test13.result">
 			<arg line="${test13.complete.dir}/test-results.xml"/>
-			<env key="RTI_HOME" path="."/><!-- required, but we set the env up properly anyway -->
-			<env key="LD_LIBRARY_PATH" path="${jdk.home.linux64}/jre/lib/amd64/server;${hla13.complete.dir}/gcc8;${cppunit.lib}"/>
+			<env key="RTI_HOME" path="/home/swuser/portico/codebase/dist/portico-2.0.5"/>
```
[the remark this answers is not in the document]
Comment (jarrodgreene) [reply codebase/profiles/linux/hla13.xml:127]: Yeah I thought I had removed any paths I temporarily hard-coded but clearly I missed this one. But it doesn't seem to work if left as `.` , not sure if that's somehow related to the updated compilers, but it seems to be overriding the environment value. I just found I can remove the line entirely and it works fine, so I'll do that if there's not another reason it has to be there

**Pull request merged**